package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@joshdb/core",
-  "version": "1.2.5",
+  "version": "1.2.6",
```

Comment:
up to @eslachance if this is worth a patch

Reply:
Yeah idk about that one, idk how NPM handles it either so i considered and added it.